RetailCoder.VBE/UI/Converters/SubractionConverter.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace Rubberduck.UI.Converters
+{
+    public class SubractionConverter : IValueConverter
```

Comment:
Is this a typo?

Reply:
yes, it is. good eye

Reply:
The typo is still present in the filename.